docs/specification.md

```diff
@@ -85,44 +86,31 @@ for more information on its content and usage.
 DAO configuration value parameters are captured by the `config` type:
 
 ```ocaml
-
-type seconds = nat
-
 type config =
-  { proposal_check : propose_params * storage -> bool
+  { proposal_check : propose_params * contract_extra -> bool
   // ^ A lambda used to verify whether a proposal can be submitted.
   // It checks 2 things: the proposal itself and the amount of tokens frozen upon submission.
   // It allows the DAO to reject a proposal by arbitrary logic and captures bond requirements
-  ; rejected_proposal_return_value : proposal * storage -> nat
+  ; rejected_proposal_return_value : proposal * contract_extra -> nat
```

Comment:
should we fix this naming mixup (ie afaiu this should be named `rejected_proposal_slash_value`) before it is too late? May be in the next release.

Reply:
Yep, thanks for the reminder, will fix before the upcoming release, opened a new issue for it: #250